packages/ui-components/style/windowedlist.css

```diff
@@ -4,17 +4,58 @@
  */
 
 .jp-WindowedPanel-outer {
+  height: 100%;
   position: relative;
-  overflow-y: auto;
+  overflow: auto;
+  transform: none;
```

Comment:
Why is transform needed here?

Reply:
It appears it isn't. I made a lot of iterations here and this is an artifact of that. Thanks!